resources/js/features/home/components/+root/CurrentlyOnline/CurrentlyOnline.tsx

```diff
@@ -51,7 +64,15 @@ export const CurrentlyOnline: FC = () => {
         </div>
 
         {currentlyOnline?.allTimeHighDate && currentlyOnline?.allTimeHighPlayers ? (
-          <p className="text-muted cursor-default italic transition hover:text-neutral-300 hover:light:text-neutral-950">
+          <p
+            className={cn(
+              isNewAllTimeHigh
+                ? 'flex items-center gap-1 text-yellow-500 light:text-yellow-700'
+                : 'text-muted cursor-default italic transition hover:text-neutral-300 hover:light:text-neutral-950',
+            )}
+          >
+            <LuTrophy className="size-4" />
```

Comment:
Your screenshots only show the trophy when it's an all time high, but this shows it all the time, and it's weirdly wrapping for me:
![image](https://github.com/user-attachments/assets/e4d2b7bc-31b0-42e5-a43f-af9ebf4985d3)

Reply:
Silly mistake on my part. Fixed in latest.